code/modules/mob/mob_movement.dm

```diff
@@ -215,6 +215,9 @@
 ///Called by client/Move()
 ///Checks to see if you are being grabbed and if so attemps to break it
 /client/proc/Process_Grab()
+	if(mob.incapacitated(FALSE, TRUE, TRUE)) // Can't break out of grabs if you're incapacitated
+		return TRUE
```

Comment:
This is going to break the ability of the code on lines 247 from running.

Reply:
The getting out of grabs? Aye that's the idea. If you mean the `move_delay` stuff. I don't think it's needed to set the `move_delay` if you don't actually move since you're stunned on the floor or cuffed etc.